test/asyncio_tests/test_examples.py

```diff
@@ -1503,3 +1503,43 @@ def MongoClient(**kwargs):
         assert isinstance(res["encrypted_unindexed"], Binary)
 
         await client_encryption.close()
+
+
+class MotorAWSLambdaExamples(AsyncIOTestCase):
+    @asyncio_test
+    async def test_shared_client(self):
+        # Start AWS Lambda Example 1
+        import os
+
+        from motor import MotorClient
+
+        client = MotorClient(host=os.environ.get("MONGODB_URI", "localhost"))
```

Comment:
Better to set a default for MONGODB_URI before the example starts so users don't mistakenly forget to set this:
```python
        os.environ.setdefault("MONGODB_URI", "localhost")
        # Start AWS Lambda Example 2
        import os
```